cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -83,6 +83,7 @@ func (command *Evaluate) Execute(args []string) (err error) {
 	// Gather models.
 	models := map[string]model.Model{}
 	{
+		availableModels := map[string]model.Model{}
```

Comment:
Now that i see this, i know the problem. Everywhere you are using languages, repositories, and models. you need to use command.*. Later on. because those are the ones cleaned up. The "languages" and "models" have all entries. the commadn.* ones are the ones cleaned up. I fixed that once before, with the categorization this bug got introduced again.

Reply:
My problem was that the `command.` contains only the IDs... but I guess I can work around that.